Dockerfile-worker

```diff
@@ -17,6 +17,9 @@ ENV PATH="/usr/local/app/bin:${PATH}"
 RUN chgrp -R 0 /usr/local/app && \
     chmod -R g=u /usr/local/app
 
+# set variable with path to Geant4 files required by TOPAS simulator
+ENV TOPAS_G4_DATA_DIR="/usr/local/app/bin/geant"
```

Comment:
Why do you locate data files in the `bin` directory ? This directory should be for binaries only.
Also - the binaries are pretty large, having very large docker images is not optimal.
Maybe better practice would be to create another docker volume, called "simulators_data" and mount it somewhere in the dockerfile worker ?

There is already a volume dedicated to storage of application between restarts, see:
https://github.com/yaptide/yaptide/blob/532ed734bfc17bf4abc1ad0afa1f27d19a8c8c08/docker-compose.yml#L11

I would avoid using this particular storage for keeping simulators data.

What about a volume called `simulators` and mounting it in `/simulators` ?
Then it may have:
  - `/simulators/geant4/bin`
  - `/simulators/geant4/data`
  - `/simulators/shieldhit12a/bin`

All `bin` directories could be added then to the PATH.

I would also avoid using `geant` and use `geant4`

Reply:
Is something like this good?
![obraz](https://github.com/yaptide/yaptide/assets/92746092/d64e204a-97cf-4774-85ac-92daf932856f)